### DISCUSSION TITLE
[Bug]: livewire powergrid table pagination css is not working

### DISCUSSION THREAD

**Discussion by MuhammadTanvir** [Power-Components/livewire-powergrid#1099] [Q&A]
### Have you searched through other issues to see if your problem is already reported or has been fixed?

Yes, I did not find it.

### Did you read the documentation?

Yes, I did not find it.

### Have you tried to publish the views?

Yes - I didn't work.

### Is there an error in the console?

No

### PHP Version

8.1

### PowerGrid

4.8

### Laravel

10

### Livewire

2.12

### Theme

Tailwind 3.x with tailwind/forms

### Describe the bug.

the powergrid table pagination button is not showing, I have attached screenshot below:

![image](https://github.com/Power-Components/livewire-powergrid/assets/16465865/abe74f61-7644-4c54-850f-c14f7580517d)

**Comment by quentindecosne** [comment]
I had the same issue and added this to tailwind.config.js

```javascript
export default {
    content: [
       ...
    ],

    presets: [
        require("./vendor/power-components/livewire-powergrid/tailwind.config.js"),
    ],


    plugins: [forms],
};
```

**Reply by MGeurts** [reply]
This solved the problem for me regarding the pagination.
**But it did not solve the issue with the double chevron-down on the "Per page" select. Any ideas?**

![Per-Page](https://github.com/Power-Components/livewire-powergrid/assets/5404867/c281122a-423c-4289-a1a8-05ddaf1fcab0)


presets: [
        require("./vendor/power-components/livewire-powergrid/tailwind.config.js"),
    ],

**Reply by EmilioVar** [reply]
I have opened an issue about it... I'll give you the solution I found:

The file causing this phenomenon is located in the following path:

/power-components/livewire-powergrid/resources/views/components/frameworks/tailwind/footer.blade.php

To fix it, I removed the element that was repeating

```
<div
                         class="pointer-events-none absolute inset-y-0 right-0 flex items-center px-2 text-pg-primary-700 dark:text-pg-primary-300">
                         <x-livewire-powergrid::icons.down class="w-4 h-4" />
                     </div>

```

After that, the element is now displayed correctly.

**Comment by luanfreitasdev** [comment]
Can you check this, ?

https://v5.livewire-powergrid.com/get-started/configure.html#_4-tailwind-settings

**Reply by MGeurts** [reply]
The link above is not working.

**Reply by luanfreitasdev** [reply]
https://livewire-powergrid.com/get-started/configure.html#tailwind-settings
